### ISSUE TITLE
Does not have any information on using VS2022 with a screen reader.

### ISSUE THREAD

**Issue by patchell** [MicrosoftDocs/visualstudio-docs#10014]
This page mentions screen reader but provides no help for getting a screen reader to read the text view'

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 378274af-75f0-8696-cc1a-5067834c6dd8
* Version Independent ID: 560d03f2-e5e8-44a2-8794-7e7bfe15f322
* Content: [Accessibility tips and tricks for Visual Studio - Visual Studio (Windows)](https://learn.microsoft.com/en-us/visualstudio/ide/reference/accessibility-tips-and-tricks?view=vs-2022)
* Content Source: [docs/ide/reference/accessibility-tips-and-tricks.md](https://github.com/MicrosoftDocs/visualstudio-docs/blob/main/docs/ide/reference/accessibility-tips-and-tricks.md)
* Service: **visual-studio-windows**
* Sub-service: **general-ide**
* GitHub Login: @j-martens
* Microsoft Alias: **jmartens**

**Comment by WilliamAntonRohm** [comment]
@patchell -- James, thank you for your feedback.  Please consider these resources:
- Ask a question or find an answer on the [Microsoft Q&A](https://learn.microsoft.com/en-us/answers/tags/176/vs) page for Visual Studio.
- Check the [Visual Studio MSDN Forum](https://social.msdn.microsoft.com/Forums/vstudio). 
- [Stack Overflow](https://stackoverflow.com/search?q=%22visual+studio%22)  
- [Reddit](https://www.reddit.com/r/VisualStudio)

@j-martens -- Josée, please look into this issue.

**Comment by anandmeg** [comment]
@patchell, thanks for your feedback. We've updated the doc to add screen readers that are supported & tested. The changes should be live later today. Let us know if you have any additional feedback. Thanks!
